flink-runtime/src/test/java/org/apache/flink/runtime/io/disk/ChannelViewsTest.java

```diff
@@ -36,16 +36,18 @@
 import org.apache.flink.runtime.operators.testutils.TestData.TupleGenerator.KeyMode;
 import org.apache.flink.runtime.operators.testutils.TestData.TupleGenerator.ValueMode;
 
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 import java.io.EOFException;
 import java.util.List;
 
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
```

Comment:
Shoud the line be repalced by `import static org.assertj.core.api.Assertions.assertThat` ?

Reply:
Thanks @RocMarshal for the hard review.
Fixed.